### DISCUSSION TITLE
Pam Jensen: Binders + Flash Drive

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1336] [Q&A]
I was talking with Pam last week, and she mentioned that she sent a binder to the lab. This binder contains some important info on the NPRB project from 2017. I have the following info about it:

- It was likely shipped to the lab at the same time as the large blue Rubbermaid totes full of samples
- It's likely dark blue
- It isn't inside any of the totes right now

That's all I've got - does anyone happen to know where this binder is? She mentioned @kubu4 might know where they are!

She also mentioned that early on, she sent @grace-ac a flash drive with an EndNote library, plus a bunch of resources (note: this is NOT the flash drive with NPRB data she sent upon her retirement). I figure it was all incorporated into Grace's PaperPile, but just wanted to double-check on that one!

**Comment by grace-ac** [comment]
I can look for the flashdrive info she gave me! I don’t have it in my paperpile- it’s on Mendeley. I can see if there’s an easy way to share, otherwise I’ll upload to google drive and send you a link. 

As for the binder… I do remember seeing it when it first arrived but can’t say where it ended up.. happy to go in and look around one of these days!

**Reply by kubu4** [reply]
Regarding binder - same response as Grace.

**Reply by sr320** [reply]
I feel like if there is one, it is blue?

**Reply by kubu4** [reply]
>  it is blue?

This is my recollection as well...
